app/src/main/java/com/sopt/now/ui/home/HomFragment.kt

```diff
@@ -0,0 +1,49 @@
+package com.sopt.now.ui.home
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import com.sopt.now.data.MultiData
+import com.sopt.now.databinding.FragmentHomeBinding
+import com.sopt.now.ui.adapter.ItemAdapter
+import com.sopt.now.ui.home.viewModel.HomeViewModel
+
+class HomeFragment : Fragment() {
+    private var _binding: FragmentHomeBinding? = null
+    private val binding
+        get() = requireNotNull(_binding) {
+            "바인딩 객체를 생성하지 않음"
+        }
+
+    private val viewModel by viewModels<HomeViewModel>()
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,
+    ): View {
+        _binding = FragmentHomeBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        val items = viewModel.friendList
+        val friendAdapter = ItemAdapter(items)
+        binding.rvFriends.adapter = friendAdapter
+        setFriendList(items)
+    }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        _binding = null
+    }
```

Comment:
```suggestion
    override fun onDestroy() {
        _binding = null
        super.onDestroy()
    }
```

이쪽이 안정성이 더 좋습니다!! 이유는 뭘까요?!

Reply:
Fragment View 의 생명주기에서 onDestroyView()로 파괴되고 나면 ViewBinding 객체를 사용해선 안된다는 주의점이 있었네요! 미리 null로 설정하고 onDestroyView() 이후에는 사용하지 않는 것이 바람직할 것 같습니다!

Reply:
민재님이 훌륭한 답변을 달아두셨네요...!!!!!! 생명주기에 따라 binding을 null처리 해주어야 한다는 사실을 이제 알았습니다.!!!!